docs/source/coredev/index.rst

```diff
@@ -22,7 +22,21 @@ Backporting Pull requests
 
 All pull requests should usually be made against ``master``, if a Pull Request
 need to be backported to an earlier release; then it should be tagged with the
-correct ``milestone``. We then use `ghpro <https://pypi.python.org/pypi/ghpro>`
+correct ``milestone``.
+
+If you are an admin on the IPython repository just mention the backport bot to
```

Comment:
I would bold the `backport bot` the first time it's used here so it stands out a bit.